proposals/lip-0049.md

```diff
@@ -492,7 +492,8 @@ registrationCCMParamsSchema = {
     "required" : [
         "name",
         "chainID",
-        "messageFeeTokenID"
+        "messageFeeTokenID",
```

Comment:
Add also `minReturnFeePerByte` in the text of line 483?

Reply:
added